LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+Servus - A small set of network oriented utilities in C++ including a zeroconf implementation.
+
+Servus is licensed under the LGPL version 3, unless noted otherwise, e.g., for external dependencies. See file COPYING.txt for the full license. External dependencies are either LGPL or BSD-licensed. See file ACKNOWLEDGEMENTS.txt and AUTHORS.txt for further details.
```

Comment:
I would prefer to relicense all files to be clear. Seems nothing major from the two "externals":

https://github.com/Eyescale/Lunchbox/commit/99b209025395bf32c66c1b8dcabc40c342079185#diff-0e416b135e5619b56a01648e287548e3
https://github.com/Eyescale/Lunchbox/commit/0875c026cfd65d9657cbc61b7ed8bf4a4e9fb653